tests/population/test_mortality.py

```diff
@@ -145,14 +147,15 @@ def test_mortality_cause_of_death(
     }
     sim.configuration.update(override_config)
     sim.setup()
+    mortality_rates = mortality.mortality_rate(sim.get_population().index)
     sim.step()
     # Only other causes and sick for cause of death
     pop1 = sim.get_population()
     for cause_of_death in ["other_causes", "sick"]:
         dead = pop1.loc[pop1["cause_of_death"] == cause_of_death]
-        # Disease model seems to set mortality rate for that diesease back to 0
+        # Disease model seems to set mortality rate for that disease back to 0
         # if a simulant dies from it
-        rates = mortality.mortality_rate(pop1.index)[cause_of_death].unique()
+        rates = mortality_rates[cause_of_death].unique()
         for mortality_rate in rates:
             if mortality_rate == 0:
```

Comment:
Can we remove this if block now?

Reply:
No because we have simualnts that are not 'sick' and simulants that are. I don't want to make the prevalence 0 to potentially mask a bug.